mkdocs.yml

```diff
@@ -45,25 +45,27 @@ markdown_extensions:
       toc_depth: 1-3
 
 plugins:
-    - search
-    # Navigation enhancements.
-    # See: https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin
-    - awesome-pages:
-        filename: .nav.yml
-    # Variables in markdown.
-    # See: https://squidfunk.github.io/mkdocs-material/reference/variables/
-    - macros
-    - redirects:
-        redirect_maps:
-          # 2022-01-14: Reorganize guides and tutorials.
-          "guides/advanced/autocomplete.md": "guides/basics/parameters/autocomplete.md"
-          "guides/advanced/dynamic-sync-tables.md": "guides/blocks/sync-tables/dynamic.md"
-          "guides/advanced/cli.md": "guides/development/cli.md"
-          "guides/advanced/libraries.md": "guides/development/libraries.md"
-          "guides/advanced/testing.md": "guides/development/testing.md"
-          "guides/troubleshooting.md": "guides/development/troubleshooting.md"
-          "get-started/cli.md": "tutorials/get-started/cli.md"
-          "get-started/web.md": "tutorials/get-started/web.md"
+  # Client-side search.
```

Comment:
FYI: The plugins content was changed from a list to a hash, in order to support the inheritance correctly, as mentioned at the bottom of [this page](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#built-in-plugins).